src/hammer-vlsi/technology/sky130/__init__.py

```diff
@@ -264,5 +269,30 @@ def sky130_power_nets(ht: HammerTool) -> bool:
                 ht.verbose_append("connect_global_net {tie} -type net -net_base_name {net}".format(tie=pwr_gnd_net.tie, net=pwr_gnd_net.name))
     return True
 
-tech = SKY130Tech()
+def sky130_connect_nets(ht: HammerTool) -> bool:
```

Comment:
Hm, this really should not need to be a hook. Question: are you using a CPF, or allowing Hammer to auto-generate the power intent? If it's the former, these global net connections should be in the CPF, If it's the latter, we really should add this `connect_global_net` here instead: https://github.com/ucb-bar/hammer-cadence-plugins/blob/4ecbce344302a5e697c2d54fd2fd80c3f70db186/par/innovus/__init__.py#L298

Reply:
Also, I think the DRC & LVS hooks should be made default (comments in ucb-bar/chipyard#1115).

Reply:
Made DRC/LVS hooks default now, and will move the sky130_power_nets hook to the auto-generated CPF in a future PR.